### PR TITLE
cephfs metrics funcational automation

### PR DESCRIPTION
**Pull request by julpark-rh**
cephfs metrics functional

[CEPH-83588303](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83588303)

https://docs.google.com/document/d/1rWk6c5RUm5P5X_NpZeM_JdjsgfakQsF4L4iGlE_2QLA/edit 
https://docs.google.com/document/d/1vD7mVaUZxhlE8hM586AboFI2bxgqdtuyzyt5q-uFkUE/edit 

Testing "counter dump" featrue

ceph tell mds.{mds_name} counter dump 2>/dev/null | jq-r'. I to_entries | map(select.key I match("mds_client_metrics")))'

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        fs_util.kernel_mount([client1], kernel_mounting_dir_1, ",".join(mon_node_ips))
+
+        # Get initial MDS metrics
+        mds_metric = fs_util.get_mds_metrics(client1, 0, fuse_mounting_dir_1)
+        log.info("Verifying opened_inodes, pinned_icaps and total_inodes")
+
+        # Initialize dictionaries to store initial inode metrics
+        inode_dic = {}
+        inode_list = ["opened_inodes", "pinned_icaps", "total_inodes"]
+        log.info(f"mds_metric_counters: {mds_metric}")
```
[the remark this answers is not in the document]
Comment (julpark-rh) [reply tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py:59]: which initial string?? can u elaborate little bit more..?

**Review by neha-gangadhar — COMMENTED** [review]
lgtm

File: tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        tc = "CEPH-83588303"
```
[the remark this answers is not in the document]
Comment (julpark-rh) [reply tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py:26]: added

File: tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (julpark-rh) [reply tests/cephfs/cephfs_metrics/cephfs_metrics_functional.py]: added

**Review by neha-gangadhar — APPROVED** [review]
Approved

**Pull request merged**